Corona-Warn-App/build.gradle

```diff
@@ -137,6 +137,14 @@ android {
         jvmTarget = "1.8"
     }
 
+    def jvmCompilerArgs = ["-Xno-kotlin-nothing-value-exception"]
+
+    tasks.withType(org.jetbrains.kotlin.gradle.dsl.KotlinJvmCompile) {
```

Comment:
Workaround for possible bug in mockk+kotlin1.4:

https://github.com/mockk/mockk/issues/490